guardian/manage-frontend
# Create generic benefits section and add to membership

## What does this change?

Refactor the existing components that list benefits into a generic component and use it for membership cards on the account overview.



## How to test

Navigate to account overview with a membership and see benefits listed under "view extras"



## How can we measure success?



## Have we considered potential risks?



## Images

![image](https://github.com/guardian/manage-frontend/assets/114918544/4714f2c2-ea1a-4ce4-ae11-397dc701cbee)



## Accessibility
